clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerRecord.java

```diff
@@ -32,7 +31,6 @@
 public class ConsumerRecord<K, V> {
     public static final long NO_TIMESTAMP = RecordBatch.NO_TIMESTAMP;
     public static final int NULL_SIZE = -1;
-    public static final int NULL_CHECKSUM = -1;
```

Comment:
Does this line need deprecation cycle also?

Reply:
Yes.